spec/rewrite/client_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require_relative '../spec_helper'
+
+describe EasyPost::Client do
+  describe 'client object' do
+    it 'create a client object timeouts' do
+      client = described_class.new(api_key: 'fake_api_key', read_timeout: 10, open_timeout: 1)
```

Comment:
issue: This test doesn't actually test that we throw a timeout error for both of the timeouts. We need to make sure that the HTTP client respects the parameters we passed to it, otherwise we are passing parameters that do nothing. That's ultimately the test I'm after here

Reply:
Added two tests to test the read and open timeouts